kernel/uid_gid.c

```diff
@@ -880,6 +881,26 @@ int check_thread_group_membership(gid_t group)
     return -1;
 }
 
+static int _path_checks(const char* pathname)
+{
+    int ret = 0;
+
+    if (!pathname)
+        return -EINVAL;
+
+    if (pathname == (void*)0xffffffffffffffff)
```

Comment:
Use we use `-1`?

Reply:
This really should be a bad address check. One of the lchown ltp tests uses `0xffffffffffffffff` as a bad address test.